documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -763,7 +667,7 @@ dataset (thereby increasing the counts four-fold).
 
 ..
    Re# 1447 should we reimplement `combine_equivalent_zones` ? See chunk symmeterize_equivalent_zones_description.bak_Re#1447 file in this folder when ticket resolved.
-   
+
```

Comment:
for previous 2 lines, put hash next to 1447 and preferably spell symmetrize correctly

Reply:
Sadly, that's the name of the file (which causes other issues on Linux), but renaming it isn't for here. This is in a comment and not visible in the rendered document.